ingenialink/canopen/network.py

```diff
@@ -631,11 +641,16 @@ def __program_control_to_stop(
         logger.info("Flashing firmware")
         with contextlib.suppress(ILError):
             servo.write(PROG_STAT_1, PROG_CTRL_STATE_STOP, subnode=0)
-        try:
+        if RUNNING_ON_WINDOWS:
```

Comment:
I don't think this is necessary.

Reply:
The thing is that the VCIError would not be defined when running on windows.